LayoutTests/imported/w3c/web-platform-tests/reporting/cross-origin-reports-isolated.https.sub.html

```diff
@@ -19,7 +19,7 @@
       // same endpoint that this frame reports to.
       await new Promise(resolve => {
         const iframe = document.createElement('iframe');
-        iframe.src = "https://{{hosts[alt][]}}:{{ports[https][0]}}/reporting/resources/csp-error.https.sub.html";
+        iframe.src = "https://{{domains[www]}}:{{ports[https][0]}}/reporting/resources/csp-error.https.sub.html";
```

Comment:
This caused the test to timeout. I think that might be because of WebKit's limited WPT domain support. Perhaps undoing this change is better.

Reply:
OK, I'll revert this test's import

Reply:
done!